index.html

```diff
@@ -0,0 +1,139 @@
+      <div class='circle-2 home-items'><a href="https://vuejs.org/" target="none">VueJS</a></div>
+      <div class='circle-3 home-items'><a href="https://www.netlify.com/" target="none">Netlify</a></div>
+    </section>
+
+    <section>
+      <main class="p-text">
+        <p class="about-me" id="first-p-tag" name="about">
+          Jacob started programming in 2006. He hasn't stopped since. He is
+          always writing code and coming up with crazy ideas. He spends most of
+          his time writing Javascript, HTML and CSS. He also enjoys writing many
+          other languages. He has written 2 books on Swift for Pearson. He has a
+          popular YouTube channel under a secret pseudonym. He has a degree in
+          piano performance and plays classical concerts frequently. Overall a
+          very nice guy.
+          </a></p>
+        <p class="about-me">
+          Jacob started at GitLab in 2015 as the first frontend engineer. He
+          went on to lead the frontend team, UX team, and lead an awesome data
+          science GitLab startup: Meltano. He has touched almost all parts of
+          GitLab in his time there. He came up with the idea for the WebIDE and
+          created the first prototype among other things.
+        </p>
+        <p class="about-me">
+          Jacob Schatz works at Netlify as the Product Engineering Manager.
+        </p>
+        <p class="about-me">
+          Jacob also speaks at conferences. He has spoken all over the world
+          about VueJS.
+        </p>
+
+        <p class="about-me">
+          Jacob also loves working on electronics and creates really fun
+          electric vehicles. His goal in life is to use gas cars less and use
+          electric vehicles more.
+        </p>
+        <p id="last-p-tag" class="about-me">
+          Jacob is a proud member of Little People of America. Although he does
+          not have dwarfism himself, he has 2 sons who both have a very rare
+          form of dwarfism that affects 1 in 500,000 people in the world. <br />
+        </p>
+      </main>
+    </section>
+    <section id="contact-container">
+
+      <form name="contact" method="POST" data-netlify="true" class="contact-form">
+        <p class="form-name" id="name_error">Name:</p>
+        <input type="text" name="username" placeholder="username"><br>
+        <p class="form-name" id="email_error">Email:</p>
+        <input type="text" name="email" placeholder="johndoe@gmail.com"><br>
+        <p class="form-name" id="message_error">Message:</p>
+        <textarea rows="10" cols="100" name="message" placeholder="Please enter text">
+        </textarea>
+        <div>
+          <input type="submit" name="submit-button" class="btn" value="Submit">
+        </div>
+      </form>
+
+    </section>
+
+  </div>
+  <section id="footer-section">
+    <p>
+      Copyright Jacob Schatz 2019
+    </p>
+  </section>
+
+  <script src="assets/jss/app.js"></script>
+</body>
+
+</html>
```

Comment:
That little red 🚫 thing means you should have an empty line for every file. It's because of this. https://stackoverflow.com/a/729725/974407

So every file in this repo (aside from binary like images) should have a new line at the end of every file.